releasenotes/notes/2.2/prepare-2.2.0-99d722b656fba0a8.yaml

```diff
@@ -0,0 +1,28 @@
+---
+prelude: >
+    The Qiskit v2.2 release further increases the functionality and versatility of the SDK
+    including enhancements for the C API and the transpiler as well as many other improvements
+    and bug fixes. Some of these additions include:
+
+      * **C API transpile function:** This version of Qiskit has added the ability for users to
+        call a version of the :func:`.transpile` function directly through the C API. The
+        :cpp:func:`qk_transpile` behaves similarly to the preset pass managers in Python,
+        with some exceptions that are not relevant in the C context. A collection of standalone
+        transpiler passes have also been added to the C API.
+      
+      * **Litinski transformation pass:** A new transformation pass was added to deal with circuits
+        with Clifford and single qubit RZ-rotation gates. The implementation is based on the transformation
+        described in `arXiv:1808.02892 <https://arxiv.org/abs/1808.02892>`__.
+
+      * **Angle bound support for targets:** We have added support for angle bounds for parameterized angle
+        rotation  within the :class:`.Target` class. When an instruction is added to the `Target` you can 
+        add the optionnal argument `angle_bounds` to specify the higher and lower bounds for parameterized
+        rotation gates. A transpiler pass :class:`.WrapAngles` was added to enforce the angle constraints.
+      
+      * **MSRV is now 1.85:** The minimum support version to buld Qiskit from source is now Rust v1.85.
+        This was done due to the need of a newer version of the Rust library `faer <https://crates.io/crates/faer>`__
+        and some errors happening in Windows.
```

Comment:
This isn't an addition, we should call this out and that this the last minor release that supports 3.9 but maybe after the bulleted list of new additions.